release-notes/major10/10.7.0/valtimo-backend-libraries.md

```diff
@@ -14,6 +14,12 @@ The following features were added:
   It is now possible to have user tasks automatically assigned to a case handler.
   Refer [to this page](/using-valtimo/document/automatic-task-assignment.md) to learn more about this feature.
 
+* **Additional status on TaakObject**
+
+  An additional status option `GESLOTEN` has been added to the 
+  [TaakStatus](https://github.com/valtimo-platform/valtimo-backend-libraries/blob/44143e0708ba32a01e9c0765a701ca2a5294da4b/zgw/portaaltaak/src/main/kotlin/com/ritense/portaaltaak/TaakObject.kt#L49)
```

Comment:
The new status is not on /main yet. But will be in the future
```suggestion
  [TaakStatus](https://github.com/valtimo-platform/valtimo-backend-libraries/blob/main/zgw/portaaltaak/src/main/kotlin/com/ritense/portaaltaak/TaakObject.kt#L49)
```

Reply:
good point 👏